src/tests/ftest/pool/RebuildTests.py

```diff
@@ -39,8 +39,7 @@
 import ServerUtils
 import WriteHostFile
 import CheckForPool
-from daos_api import DaosContext, DaosPool, DaosServer, DaosContainer
-from daos_api import DaosApiError
+from daos_api import DaosContext, DaosPool, DaosServer, DaosContainer, DaosApiError
```

Comment:
(pylint-line-too-long) Line too long (83/80)

Reply:
no belong to current patch.